### ISSUE TITLE
UX: I want tab from stage to move to commit box

### ISSUE THREAD

**Issue by benogle** [atom/github#856]
I know I can probably remap this, but It seems a little weird that hitting tab from unstaged changes, moves to the staged files, and not directly to the commit box, but only if there are things staged. So it may take 1 or 2 tab presses to get to the commit box. e.g.

* Have some unstaged changes, and nothing staged
* tab once
* Commit box selected

+

* Stage all your changes
* tab once
* Commit box selected

And 

* Have some unstaged changes, and staged changes; focus is on unstaged
* tab _twice_
* Commit box selected

**Comment by BinaryMuse** [comment]
Agreed about the tab weirdness. FWIW you can use the `github:commit` keybind (cmd/ctrl-enter by default) to jump to the commit box from anywhere in the staging view.

**Comment by benogle** [comment]
Word thanks, I will remap for now.

**Comment by smashwilson** [comment]
:+1: Messing with keybindings post-release always makes me a bit nervous but this extra bit of friction is bugging me, too.

**Comment by AmeerAnsari** [comment]
great work ....

**Comment by kuychaco** [comment]
Hmm yeah I can see that the inconsistent behavior might seem funky. That said, personally I expect hitting `tab` to take me to the next reasonable thing, whatever it is. If we changed the `tab` behavior when the Unstaged Changes list is focused to always focus the commit box, then what would we offer by way of keyboard navigation for quickly getting to the Staged Changes list?

**Comment by benogle** [comment]
> then what would we offer by way of keyboard navigation for quickly getting to the Staged Changes list

The arrow keys, I think?

Or it could be that the staged / unstaged boxes can receive focus even when they dont have items? I would be fine with 2 tabs if it were always 2 tabs no matter the items.
